src/Catty/Defines/NetworkDefines.swift

```diff
@@ -97,4 +97,8 @@ class NetworkDefines: NSObject {
 
     @objc static let kUserIsLoggedIn = "userIsLoggedIn"
     @objc static let kUserLoginToken = "userLoginToken"
+
+    // MARK: WebRequestDownloader
+
+    static let kWebRequestMaxDownloadSize = 1000000 // = 1MB
```

Comment:
Please add `InBytes` as a postfix to clarify in which unit the size limit is (in case the comment gets removed and so that it is clear in all places without having the comment).